.gitignore

```diff
@@ -32,3 +32,14 @@ keyring
 
 # Ignore intelliJ files
 .idea
+.settings/org.eclipse.buildship.core.prefs
+.settings/org.eclipse.jdt.core.prefs
+tests/testfunctions/.classpath
+tests/testfunctions/gradle/
+.classpath
+.vscode/
+.project
+tests/testfunctions/bin/
+tests/testfunctions/.serverless/
+tests/testfunctions/.settings/
+tests/testfunctions/gradle/wrapper/gradle-wrapper.properties
```

Comment:
You have 'tests/testfunctions/gradle/wrapper/gradle-wrapper.properties' commited to the repo. Is that intentional?